loopd/daemon.go

```diff
@@ -103,7 +103,13 @@ func daemon(config *config) error {
 	}
 	defer restListener.Close()
 	proxy := &http.Server{Handler: mux}
-	go proxy.Serve(restListener)
+
+	go func() {
+		err := proxy.Serve(restListener)
+		if err != nil && err != http.ErrServerClosed {
```

Comment:
It may be worth adding a comment on why `ErrServerClosed` is ignored.

Reply:
done: https://github.com/lightninglabs/loop/pull/115/commits/aaf13621e5aef471d73d0abd572d80a86f895ffe

Reply:
:+1: